packages/driver/cypress/fixtures/dropdown.html

```diff
@@ -1,44 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Did we lose test coverage at some point ? did you look into when these became obsolete?

Reply:
Maybe my Git-fu is weak, but I was unable to tell when these files were ever being used for testing. They are over 5 years old.